# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/html/HTMLIFrameElement.cpp
# Commit: b326d33f98cb
# Full Hash: b326d33f98cbc1d08d8ca10609ae24e8c852767c
# Author: Andreas Farre <farre@mozilla.com>
# Date: 2020-08-01 09:48:10
# Regressor Bug: 1613431
# File Overlap Count: 1
# Description:
#   Bug 1613431 - Part 2: Ignore synced setters return value. r=nika
#   
#   Depends on D83645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D83646
# ==============================================================================

diff -r 4d29677fb419 -r b326d33f98cb dom/html/HTMLIFrameElement.cpp
--- a/dom/html/HTMLIFrameElement.cpp	Fri Jul 31 13:36:45 2020 +0000
+++ b/dom/html/HTMLIFrameElement.cpp	Fri Jul 31 13:37:00 2020 +0000
@@ -279,7 +279,8 @@
     return;
   }
 
-  browsingContext->SetFeaturePolicy(mFeaturePolicy);
+  // Return value of setting synced field should be checked. See bug 1656492.
+  Unused << browsingContext->SetFeaturePolicy(mFeaturePolicy);
 }
 
 already_AddRefed<nsIPrincipal>